urbanairship-location-react-native/package.json

```diff
@@ -40,7 +40,7 @@
   "scripts": {
     "prepare": "bob build",
     "test": "jest --verbose",
-    "generate-docs": "typedoc --out docs --exclude js/__tests__ js/"
+    "generate-docs": "typedoc --out docs --exclude **/index.ts"
```

Comment:
I'm excluding `index.ts` because it just exports the other methods and properties, which cluttered up the docs.